config/webpack.common.js

```diff
@@ -0,0 +1,73 @@
+const paths = require("./paths");
+
+const { CleanWebpackPlugin } = require("clean-webpack-plugin");
+const CopyWebpackPlugin = require("copy-webpack-plugin");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+
+module.exports = {
+  // Where webpack looks to start building the bundle
+  entry: [paths.src + "/index.js"],
+
+  // Where webpack outputs the assets and bundles
+  output: {
+    path: paths.build,
+    filename: "[name].bundle.js",
+    publicPath: "/",
+  },
+
+  // Customize the webpack build process
+  plugins: [
+    // Removes/cleans build folders and unused assets when rebuilding
+    new CleanWebpackPlugin(),
+
+    // Copies files from target to destination folder
+    new CopyWebpackPlugin({
+      patterns: [
+        {
+          from: paths.public,
+          to: "assets",
+          globOptions: {
+            ignore: ["*.DS_Store"],
+          },
+          noErrorOnMissing: true,
+        },
+      ],
+    }),
+
+    // Generates an HTML file from a template
+    // Generates deprecation warning: https://github.com/jantimon/html-webpack-plugin/issues/1501
+    new HtmlWebpackPlugin({
+      title: "webpack Boilerplate",
+      template: paths.src + "/template.html", // template file
```

Comment:
> [@majac91] Edit: I figured out how to integrate images but now there's a conflict with carousel.js that I'm not sure how to solve. Another thing is _flaticon.scss, I added "import ./icons/font/_flaticon.scss"; in index.js but the icons still don't display.

When using webpack, we don't need to add js/css assets to `template.html`, because they are defined here in this webpack config. So you can remove all of them (flaticon, and index.js).

The icons font not working... I couldn't discover why 🤔 Maybe it's easier if you could use an external font URL like it's done for google fonts.

Reply:
Okay, I'll replace the icons. Any idea what's causing the carousel issue?

Reply:
I didn't notice any issue with the carousel. Can you explain it in detail?

Reply:
I'll send you a loom

Reply:
Here's a loom https://www.loom.com/share/d2b01b82886e472d9d7710456c2e2cea

Reply:
Regarding icons, I was able to fix the static but I'm not sure what to do about the dynamic ones which had the icon in a pseudo-element (```.gallery__edit-btn```, ```.overlay__close```, ```.overlay__delete```). I've tried adding them in variables as background but either it doesn't work or I didn't do it right. 

The code is becoming less readable too with all these extra lines that weren't necessary with Parcel,  there must be a better way to organize it?

Reply:
Yeah, with webpack (and other bundles that you'll deal in the future) require to explicitly tell the paths for the needed files. Maybe this helps you: https://stackoverflow.com/questions/42118296/dynamically-import-images-from-a-directory-using-webpack

Reply:
Regarding the carousel bug, a quick debug in the DOM, it seems like the last image does not have a source. 
![image](https://user-images.githubusercontent.com/14869087/102147418-9f952180-3e62-11eb-93a0-779af6c8f744.png)

I believe you'll find what's causing the issue with your bff `console.log()` and using breakpoints ;)

Reply:
I'm looking and I'm not seeing it. The first and the last clone didn't have paths, that much I figured but no clue what's different about slide 4 in particular.

Reply:
Also this slipped throught the cracks 

> Regarding icons, I was able to fix the static but I'm not sure what to do about the dynamic ones which had the icon in a pseudo-element (`.gallery__edit-btn`, `.overlay__close`, `.overlay__delete`). I've tried adding them in variables as background but either it doesn't work or I didn't do it right.